client/src/hooks/index.ts

```diff
@@ -1,5 +1,21 @@
 import {TypedUseSelectorHook, useDispatch, useSelector} from 'react-redux';
 import {RootState, AppDispatch} from "../store/store";
+import { useEffect, useState } from "react";
 
 export const useAppDispatch = () => useDispatch<AppDispatch>();
-export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector;
\ No newline at end of file
+export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector;
```

Comment:
do not put all hooks to one file these two are main for redux purpose so they can be here but each subsequent one make as own file useViewport.ts for example